### PR TITLE
operator cc-operator (0.7.0)

### PR DESCRIPTION
**Pull request by wainersm**
Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [ ] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [ ] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [ ] Have you tested your Operator with all Custom Resource Definitions?
* [ ] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [ ] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [ ] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [ ] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [ ] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [ ] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description about the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

File: operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+              }
+            },
+            "runtimeName": "kata"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.23.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: cc-operator.v0.0.1
```
Comment (jensfr) [inline operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml:244]: this needs to be changed

File: operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,474 @@
...
+    name: Fabiano
+  - email: jensfr
+    name: Jens
+  - email: bpradipt
+    name: Pradipta
+  maturity: alpha
+  provider:
+    name: Confidential Containers Community
+    url: https://github.com/confidential-containers
+  version: 0.0.1
```
Comment (jensfr) [inline operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml:474]: this needs to be 0.7.0 and you need to add
`  replaces: cc-operator.v0.5.0`  (same indentation as 'version')

maintainers should be 
`  maintainers:
  - email: cncf-ccontainers-maintainers@lists.cncf.io
    name: Jens Freimann
  - email: cncf-ccontainers-maintainers@lists.cncf.io
    name: Pradipta Banerjee
  - email: cncf-ccontainers-maintainers@lists.cncf.io
    name: Fabiano Fidencio
`

File: operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (wainersm) [reply operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml]: Done

File: operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (wainersm) [reply operators/cc-operator/0.7.0/manifests/cc-operator.clusterserviceversion.yaml]: Done

**Comment by jensfr** [comment]
@wainersm I think you need to set metadata.annotations.categories to "Security" to make the test pass

**Comment by jensfr** [comment]
@wainersm I think you should copy the spec.description section including .icon and copy it to the new 0.7.0 CSV

**Comment by jensfr** [comment]
minKubeVersion should be 1.20

**Comment by mporrato** [comment]
Closing as newer bundles have already been published for this operator.

**Pull request closed without merging**